diff -urN xfprint-4.6.0/configure.in xfprint-4.6.0.new/configure.in
--- xfprint-4.6.0/configure.in	2009-02-25 07:36:35.000000000 +0100
+++ xfprint-4.6.0.new/configure.in	2009-03-01 00:54:08.000000000 +0100
@@ -135,7 +135,7 @@
 AC_HELP_STRING([--enable-bsdlpr], [Enable BSD-LPR support in xfprint]),
 [], [enable_bsdlpr=yes])
 if test x"$enable_bsdlpr" = x"yes"; then
-  AC_CHECK_PROG([bsdlpr_found], [lpr], [yes], [no])
+  bsdlpr_found="yes"
 else
   bsdlpr_found="no"
 fi
