--- xfprint-4.1.91/configure.ac.orig	2004-10-29 22:40:40.000000000 +0200
+++ xfprint-4.1.91/configure.ac	2004-11-13 21:36:48.555924624 +0100
@@ -33,7 +33,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfprint], [ar az be bg ca cs de es es_MX et eu fa fi fr gu he hi hu
-		    it ja ko lt ms nl ru sk sv ta tr pa pl pt_BR pt_PT uk vi
+		    it ja ko lt ms nl ru sk sv ta tr pa pl pt_BR pt uk vi
 		    zh_CN zh_TW])
 
 dnl Check for gmodule >= 2.2.0
