diff -ur xfprint.orig/configure.ac xfprint/configure.ac
--- xfprint.orig/configure.ac	2004-07-27 22:21:06.000000000 +0200
+++ xfprint/configure.ac	2004-08-07 02:53:06.800787512 +0200
@@ -31,8 +31,8 @@
 AC_FUNC_MMAP
 
 dnl Check for i18n support
-BM_I18N([xfprint], [ar az bg ca cs de es es_MX eu fa_IR fr hi hu it ja ko lt ms nl ru
-	 sv ta tr pl pt_BR pt_PT uk vi zh_CN zh_TW])
+BM_I18N([xfprint], [ar az bg ca cs de es es_MX eu fa fr hi hu it ja ko lt ms nl ru
+	 sv ta tr pl pt_BR pt uk vi zh_CN zh_TW])
 
 dnl Check for GThread >= 2.0 installed
 BM_DEPEND([GTHREAD], [gthread-2.0], [2.0.6])
