--- xfprint-4.2.2/configure.ac.orig	2005-05-15 10:35:39.000000000 +0200
+++ xfprint-4.2.2/configure.ac	2005-07-12 16:59:42.920871848 +0200
@@ -48,7 +48,7 @@
 dnl Check for i18n support
 BM_I18N([xfprint], [ar az be bg bn_IN ca cs de el en_GB es es_MX et eu fa
 		    fi fr gu he hi hu it ja ko lt mr ms nl ru sk sv ta tr pa
-		    pl pt_BR pt_PT sk uk vi zh_CN zh_TW])
+		    pl pt_BR pt sk uk vi zh_CN zh_TW])
 
 dnl Check for gmodule >= 2.2.0
 BM_DEPEND([GMODULE], [gmodule-2.0], [2.2.0])
