diff -urN xfprint-4.4.2/configure.in xfprint-4.4.2.new/configure.in
--- xfprint-4.4.2/configure.in	2007-11-17 20:55:04.000000000 +0100
+++ xfprint-4.4.2.new/configure.in	2007-12-07 16:59:56.000000000 +0100
@@ -80,7 +80,7 @@
 AC_SUBST([PRINTCAP_PATH])
 
 dnl Check for i18n support
-XDT_I18N([am ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt lv mk mr ms nb_NO nl ro ru sk sv ta tr pa pl pt_BR pt_PT uk vi zh_CN zh_TW])
+XDT_I18N([am ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt lv mk mr ms nb nl ro ru sk sv ta tr pa pl pt_BR pt uk vi zh_CN zh_TW])
 
 dnl Check for gmodule
 XDT_CHECK_PACKAGE([GMODULE], [gmodule-2.0], [gtk_minimum_version])
diff -urN xfprint-4.4.2/po/LINGUAS xfprint-4.4.2.new/po/LINGUAS
--- xfprint-4.4.2/po/LINGUAS	2007-11-17 20:32:01.000000000 +0100
+++ xfprint-4.4.2.new/po/LINGUAS	2007-12-07 16:59:41.000000000 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-am ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt lv mk mr ms nb_NO nl ro ru sk sv ta tr pa pl pt_BR pt_PT uk vi zh_CN zh_TW
+am ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt lv mk mr ms nb nl ro ru sk sv ta tr pa pl pt_BR pt uk vi zh_CN zh_TW
